pkgs/top-level/all-packages.nix

```diff
@@ -10047,6 +10047,15 @@ let
       ];
   };
 
+  linux_4_4 = callPackage ../os-specific/linux/kernel/linux-4.4.nix {
+    kernelPatches = [ kernelPatches.bridge_stp_helper ]
+      ++ lib.optionals ((platform.kernelArch or null) == "mips")
+      [ kernelPatches.mips_fpureg_emu
```

Comment:
I do not :-)  they seem to have been copied from kernel to kernel, so I kept with the tradition ;-)